Source/WebCore/page/Quirks.cpp

```diff
@@ -2971,7 +2983,8 @@ void Quirks::determineRelevantQuirks()
         { "zomato"_s, &handleZomatoQuirks },
 #endif
         { "zoom"_s, &handleZoomQuirks },
-        { "dailymail"_s, &handleDailyMailCoUkQuirks }
+        { "dailymail"_s, &handleDailyMailCoUkQuirks },
+        { "capitalgroup"_s, &handleCapitalGroupQuirks }
```

Comment:
I'm not the best reviewer for this, but it appears this list is meant to be in chronological order, but these last two entries break that. (Not sure if there's an easy way to assert that.)

Reply:
I moved it up.